cypress/e2e/uploader_spec.cy.ts

```diff
@@ -0,0 +1,26 @@
+describe('template spec', () => {
+  it('passes', () => {
+    // TODO: Get SSL set up.
+    cy.visit('http://localhost:3000/')
+    let title = `TestZine ${Date.now()}`
+    const allZines = '[data-test="zine"]'
+    cy.get(allZines).should($z => {
+      expect($z).to.not.contain(title)
+    })
```

Comment:
I found out `cy.contains('Anarchy 300').should('not.exist')` wasn't actually working